styles.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #07535b;
```

Comment:
backgroundColor should not use this part because this might have unintended consequences on certain element .